modules/osdk-csv-bundle-files.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-csv-bundle-files_{context}"]
+= Generated files and resources
+
+The `make bundle` command creates the following files and directories in your Operator project:
+
+* A bundle manifests directory `bundle/manifests` containing a `ClusterServiceVersion` object
+* A bundle metadata directory `bundle/metadata`
+* All custom resource definitions (CRDs) in `config/crd`
+* A Dockerfile `bundle.Dockerfile`
```

Comment:
Should the directory name have commas in these three bullets?
`A bundle manifests directory, `bundle/manifests`, containing `